prometheus/histogram.go

```diff
@@ -1575,3 +1606,149 @@ func addAndResetCounts(hot, cold *histogramCounts) {
 	atomic.AddUint64(&hot.nativeHistogramZeroBucket, atomic.LoadUint64(&cold.nativeHistogramZeroBucket))
 	atomic.StoreUint64(&cold.nativeHistogramZeroBucket, 0)
 }
+
+type nativeExemplars struct {
+	ttl time.Duration
+
+	exemplars []*dto.Exemplar
+
+	sync.Mutex
```

Comment:
Nit: I see no reason to use a blank line between exemplars and ttl. With only three members, one might even argue that the mutex shouldn't be separated.

I have a minor preference to list a mutex that protects the whole struct first, but I'm not sure if that's really a convention we generally follow in the code base.

In summary:

```suggestion
	sync.Mutex

	ttl       time.Duration
	exemplars []*dto.Exemplar
```